### DISCUSSION TITLE
Get dropdown controller value from watch

### DISCUSSION THREAD

**Discussion by kithma** [react-hook-form/react-hook-form#5927] [Q&A]
when the form loading I use setValue to set a value to my dropdown controller, however, I cannot catch that value using `watch` Is there any way I could achieve this? and why is it return undefined for watch even after I set the value

```
const watchItem = watch("orderDetails.item");

useEffect(() => {
                const item = "abc"
		setValue("orderDetails.item", item)
}, [catStore])
```

**Comment by bluebill1049** [comment]
can you share a codesandbox?
